controllers/volumereplicationgroup_controller.go

```diff
@@ -197,19 +201,34 @@ func (v *VolumeReplicationGroupReconciler) Reconcile(ctx context.Context, req ct
 
 	pvcList, err := v.HandlePersistentVolumeClaims(ctx, volRepGroup)
 	if err != nil {
-		log.Error("Handling of Persistent Volume Claims of application failed: ", volRepGroup.Spec.ApplicationName)
+		log.Error(err, "Handling of Persistent Volume Claims of application failed")
 
 		return ctrl.Result{}, err
 	}
 
-	err = v.HandlePersistentVolumes(ctx, pvcList)
-	if err != nil {
-		log.Error("Handling of Persistent Volumes for PVCs of application failed: ", volRepGroup.Spec.ApplicationName)
+	requeue := false
 
-		return ctrl.Result{}, err
+	for _, pvc := range pvcList.Items {
+		volRep := &volrep.VolumeReplication{}
+
+		err = v.Get(ctx, types.NamespacedName{Name: pvc.Name, Namespace: pvc.Namespace}, volRep)
+		if err != nil {
+			if errors.IsNotFound(err) {
+				if err = v.createVolumeReplicationCRForPVC(ctx, pvc); err == nil {
+					continue
+				}
+			}
+
+			// requeue on failure to ensure any PVC not having a corresponding VR CR
+			requeue = true
```

Comment:
I wanted these statements cuddled, IOW like,
```
			// requeue on failure to ensure any PVC not having a corresponding VR CR
			requeue = true
			log.Error(err, "Failed to get or create VolumeReplication CR for PVC", pvc.Name, pvc.Namespace)

			continue
```

This is the [rule](https://github.com/bombsimon/wsl/blob/master/doc/configuration.md#allow-assign-and-call) that disallows it.

Wanted to check if this rule in the `wsl` linter can be disabled, as it seems too strict and IMO causes a lot of line spaces in the code.

Reply:
Attempted to leverage the setting, but did not prove useful. Other settings for the wsl linter make it too generous. Ended up tightening another setting instead here https://github.com/RamenDR/ramen/pull/16